src/convertToObject.js

```diff
@@ -5,8 +5,17 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
+function convertToObject(styleStr) {
+  const styleObj = {};
+  const styleArr = styleStr.split(';');
+
+  styleArr.forEach((style) => {
+    const [key, value] = style.split(':').map((s) => s.trim());
+
+    styleObj[key] = value;
+  });
```

Comment:
Consider adding a check to ensure that both `key` and `value` are not undefined or empty. This will prevent adding invalid entries to the `styleObj` when the style string contains empty segments or malformed key-value pairs.